### PR TITLE
[IMP] web: revamp Graph view

### PR DESCRIPTION
**Pull request by Brieuc-brd**
The purpose is to simplify the SCSS of the Graph View.

Some changes was also made to be able to use a bolder font weight (700) in some cases in Enterprise and to keep the consistency in the current design. 

Requires:
- https://github.com/odoo/enterprise/pull/23633

task-2740030

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋

File: addons/web/static/src/views/graph/graph_view.scss [hunk 1]
```diff
@@ -1,83 +1,7 @@
...
-
-  .o_graph_canvas_container {
-    padding-top: 5px;
-    position: relative;
-    height: 100%;
-    width: 100%;
-    top: 0px;
-    left: 0px;
+.o_graph_view > .o_content > .o_renderer .o_graph_canvas_container {
+    min-height: 250px;
```
Comment (stefanorigano) [inline addons/web/static/src/views/graph/graph_view.scss:2]: I really would like to get rid of this ^^
Why is it necessary?

File: addons/web/static/src/views/graph/graph_view.scss [hunk 2]
```diff
@@ -1,83 +1,7 @@
...
+    
     canvas {
```
Comment (stefanorigano) [inline addons/web/static/src/views/graph/graph_view.scss:4]: There are canvas rendered in the graph view that shouldn't use `$o-view-background-color`?
If not (I guess so) simplify the selector: the resulting one is probably unnecessarily specific. 
```css
.o_graph_view > .o_content > .o_renderer .o_graph_canvas_container canvas {}
```

File: addons/web/static/src/views/graph/graph_view.scss (hunk 1 above)
Comment (Brieuc-brd) [reply addons/web/static/src/views/graph/graph_view.scss]: @stefanorigano I would also like to 🙈 , but without the `min-height`, the graph can't be readable when the height of the viewport is too tiny. With this `min-height`, the canvas is still readable with a scroll bar at least.

Capture d’écran 2022-01-26 à 15 56 54

With the API of BS5 we could create some utils classes to manage the min-height.

**Comment by Brieuc-brd** [comment]
Hello @stefanorigano 👋 ,
I finally moved the `min-height` in the `dashboard_view.scss` in Enterprise. There is now only one property in the scss file.
What do you think ?

**Comment by stefanorigano** [comment]
good for me 👍

**Comment by Brieuc-brd** [comment]
@aab-odoo , ready for tech. review :)
Cc. @adr-odoo

**Review by aab-odoo — APPROVED** [review]

**Comment by Brieuc-brd** [comment]
The branch has been rebased to resolve conflicts

**Pull request closed without merging**